### ISSUE TITLE
Clean up navbar styling

### ISSUE THREAD

**Issue by Mayamariaruth** [NoroffFEU/edu-gate#113]
## Tasks
- Make sure navbar texts are following style guide
- Make sure navbar texts are all the same (size, font, color etc)

**Comment by Mayamariaruth** [comment]
Parent ticket #59

**Comment by Martinekong** [comment]
The nav bar on desktop was Poppins semiBold 24px (equals to H2 in the style guide) - I changed it to 20px (equals to H3 in the style guide). 

The dropdown menu was Roboto regular 28px - I changed it to Poppins 20px to match the nav bar, but let it remain as regular so it is a little different from the navigation that it drops down from. 

An alternative is to make the nav bar **bold** and the dropdown menu **semiBold** since **regular** isn’t in the style guide. 

I cant check the hamburger menu on mobile yet since it is not completed.
